mslib/mscolab/sockets_manager.py

```diff
@@ -51,12 +51,33 @@ def __init__(self, chat_manager, file_manager):
         """
         super(SocketsManager, self).__init__()
         self.sockets = []
+        self.active_users_per_operation = {}
         self.cm = chat_manager
         self.fm = file_manager
 
     def handle_connect(self):
         logging.debug(request.sid)
 
+    def handle_operation_selected(self, json_config):
+        logging.debug("Operation selected: {}".format(json_config))
+        token = json_config['token']
+        op_id = json_config['op_id']
+        user = User.verify_auth_token(token)
+        if not user:
```

Comment:
if user is None:
   return None ?

Reply:
In other functions, it was written like this only so I just wrote this like them. Should I return None?

Reply:
None is the default for return.

I meant the condition check before. It is also on other places wrong when it is done similiar.

The C based python allows currently this but from the definition it better should compare by "is" instead of abusing boolean.

Another python Implementation may use the specification different.

Reply:
I see, I will correct this in other places too